docs/ldml/tr35.md

```diff
@@ -884,13 +884,13 @@ LDML inherits time zone IDs from the tz database [[Olson](#Olson)]. Because thes
 
 The short identifiers use UN/LOCODE [[LOCODE](#LOCODE)] (excluding a space character) codes where possible. For example, the short identifier for "America/Los_Angeles" is "uslax" (the LOCODE for Los Angeles, US is "US LAX"). Identifiers of length not equal to 5 are used where there is no corresponding UN/LOCODE, such as "usnavajo" for "America/Shiprock", or "utcw01" for "Etc/GMT+1", so that they do not overlap with future UN/LOCODE.
 
-Although the first two letters of a short identifier may match an ISO 3166 two-letter country code, a user should not assume that the time zone belongs to the country. The first two letters in an identifier of length not equal to 5 has no meaning. Also, the identifiers are stabilized, meaning that they will not change no matter what changes happen in the base standard. So if Hawaii leaves the US and joins Canada as a new province, the short time zone identifier "ushnl" would not change in CLDR even if the UN/LOCODE changes to "cahnl" or something else.
+Although the first two letters of a short identifier may match an ISO 3166 two-letter country code, a user should not assume that the time zone belongs to the country. The first two letters in an identifier of length not equal to 5 have no meaning. Also, the identifiers are stabilized, meaning that they will not change no matter what changes happen in the base standard. So if Hawaii leaves the US and joins Canada as a new province, the short time zone identifier "ushnl" would not change in CLDR even if the UN/LOCODE changes to "cahnl" or something else.
```

Comment:
`The first two letters in an identifier of length not equal to 5 have no meaning.` seems confusing to me.

Does this mean, `The first two letters in an identifier have no meaning when they are in a short identifier with a length shorter than 5 letters.`? I wasn't sure what the significance of the short identifier having a 5 character length indicates. Or does this mean something else?

Reply:
- A short identifier of exactly 5 characters is a UN/LOCODE.  So `uslax` is US + LAX (airport). That's the namespace, and the LOCODE always starts with the 3166 country name.
- `utcw01` doesn't follow this, `UT` is not a country code. 
- `usnavajo` has a country code arbitrarily added. It could have been `navajo` (6) as `zznavajo` is too long.

So no it doesn't mean less than five, but less than five OR greater than 5.